superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterControls/FilterControls.tsx

```diff
@@ -42,6 +42,9 @@ import { buildCascadeFiltersTree } from './utils';
 
 const Wrapper = styled.div`
   padding: ${({ theme }) => theme.gridUnit * 4}px;
+  // 108px padding to make room for buttons with position: absolute
```

Comment:
If we change to a white background then could move the bottom buttons outside the scrollable area.